Seccion 41 (COMPLETO)

CarlosJCD opened the pull request:
s avances en codigo de la seccion #41 del curso.

CarlosJCD committed:
Video #403 - Seccion #41 - Adaptando index de Admin en MVC

CarlosJCD committed:
Video #404 - Seccion #41 - Formulario de Crear propiedad MVC

CarlosJCD committed:
Video #405 - Seccion #41 - Asociando funciones al metodo POST

CarlosJCD committed:
Video #406 - Seccion #41 -  Finalizando crear propiedades

CarlosJCD committed:
Video #407 - Seccion #41 - Formulario actualizar propiedad

CarlosJCD committed:
Video #408 - Seccion #41 - Finalizando actualizar propiedad

CarlosJCD committed:
Video #409 - Seccion #41 - Eliminar propiedad en MVC

CarlosJCD committed:
Video #410 - Sección #41 - Mostrar vendedores en index de admin MVC

CarlosJCD committed:
Video #411 - Seccion #41 - Creando el controlador de Vendedor

CarlosJCD committed:
Video #412 y #413 - Seccion #41 - Adaptando crear vendedores en MVC

CarlosJCD committed:
Videos #414 y #415 - Seccion #41 - Adaptando actualizar vendedor en MVC

CarlosJCD committed:
Video #416 - Seccion #41 - Adaptando eliminar vendedor a MVC

CheloChidoone reviewed (changes requested):
@CarlosJCD, no funciona bien tu actualizar propiedades cuando quiero cambiar la imagen no la esta actualizando, revisa eso y también recomendaría que en este mismo PR agregues tus avances hasta que termines la sección 41 porque ahora se ve incompleto aun tu proyecto ya que no has terminado

CarlosJCD committed:
Correccion: Actualizar imagen en Actualizar Propiedad

CarlosJCD committed:
Borrando imagenes sobrantes de la carpeta "imagenesPropiedad"

CarlosJCD committed:
Video #417 - Seccion #41 - Controlador para la pagina principal

CarlosJCD committed:
Video #418 - Seccion #41 - Incluyendo pagina index

CarlosJCD committed:
Video #419 - Seccion #41 - Incluyendo pagina "Nosotros"

CarlosJCD committed:
Video #420 - Seccion #41 - Incluyendo pagina "Propiedades"

CarlosJCD committed:
Video #421 - Seccion #41 - Incluyendo pagina "Propiedad"

CarlosJCD committed:
Video #422 - Seccion #41 (FIN) - Incluyendo pagina "Blog" (estatico)

CarlosJCD committed:
Extra: Seccion "blog" con entradas dinamicas

Cambiando la seccion del blog a entradas dinamica, con un nuevo modelo llamado "EntradaBlog", una nueva seccion en el index de admin para las operaciones de CRUD, un controlador y una tabla en la base de datos. (El archivo de la base de datos actualizada se encuentra en la carpeta "database")
